pkgs/tools/networking/dublin-traceroute/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, jsoncpp
+, libtins
+, libpcap
+, openssl
+, nixosTests
+}:
```

Comment:
These have a guideline about order they should follow (last point): https://nixos.org/manual/nixpkgs/stable/#sec-syntax